### DISCUSSION TITLE
Model not loading in oculus quest vr headet (gltf runtime import)

### DISCUSSION THREAD

**Discussion by gs-sangeeth** [atteneder/glTFast#441] [Q&A]
Was developing a metaverse scene for Oculus Quest device which required importing gltf models at runtime from an API.
The model is getting loaded correctly when testing out in the Unity Editor. Also, the apk build works in android smartphones (the model is getting loaded), but in an oculus quest 2, the model isn't getting loaded.

Do any of you guys know why this might be happening, or provide a solution for this?

**Comment by atteneder** [comment]
Hi @gs-sangeeth ,

Unfortunately I've never tested on a Quest. I'd recommend:

- Gather more information (what does the logs say? can you debug? If yes, how far in the process does it go?
- Try different assets (gltf vs. glb; minimalistic ones). Do all of them fail?
- Try loading locally from StreamingAssets instead of HTTP. Does that work?
- Lastly: Ask in Quest specific Unity forums. Maybe the platform has some differences we're not aware of

hth

**Comment by giulianamuto** [comment]
It works for me. it's a shader problem, set them well!
